jupyterhub/templates/hub/deployment.yaml

```diff
@@ -54,11 +54,13 @@ spec:
         # change this
         - name: SINGLEUSER_IMAGE
           value:  "{{ .Values.singleuser.image.name }}:{{ .Values.singleuser.image.tag }}"
+        {{ if ne .Values.hub.cookieSecret null }}
```

Comment:
This should just be `{{ if .Values.hub.cookieSecret }}`. `ne` in go templates doesn't work with `nil` - it's pretty terrible.